packages/fiori/src/UploadCollectionItem.js

```diff
@@ -0,0 +1,380 @@
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
```

Comment:
if you don't have <ui5-icon tag in UploadCollectionitem.hbs, you don't need to import Icon.js

Reply:
Removed it